apps/vr-tests/src/stories/TextField.stories.tsx

```diff
@@ -49,3 +49,35 @@ storiesOf('TextField', module)
       rtl: true
     }
   );
+
```

Comment:
I don't think this screener test is actually necessary. There is already a suffix / prefix test to catch the style of things. For a real test that would catch focus, you need to do enzyme testing.

Reply:
I'm afraid I know nothing of enzyme, the intent of this test was that it clicks the prefix/suffix so you can verify it ended up in a focused state. But like I said, if it's not worth keeping feel free to throw it out, I don't mind.

Reply:
Yeah, go ahead and delete that test for now. I think we can backfill with some tests if necessary. That will get you to pass the tests, etc.

Reply:
Sure, I'll trim it out tomorrow.